code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -173,7 +173,7 @@
 
 	for(var/obj/item/organ/external/EO in organs)
 		if(EO.brute_dam || EO.burn_dam)
-			output += "[EO.name] - <span class='warning'>[EO.burn_dam + EO.brute_dam > ROBOLIMB_REPAIR_CAP ? "Heavy Damage" : "Light Damage"]</span>\n"
+			output += "[EO.name] - <span class='warning'>[EO.burn_dam + EO.brute_dam > EO.min_broken_damage + 10 ? "Heavy Damage" : "Light Damage"]</span>\n" //VOREstation edit - Makes robotic limb damage scalable
```

Comment:
I meannnn. I'm not okay with you changing how robolimb repair works for people who don't have this trait, in a PR titled "Adds hollow bones trait" and doesn't mention this change in the description, just from a general perspective.

Definitely not this one, since this change makes synths without this trait far more robust, raising things like their torso repair cap from 30 to 45.